### PR TITLE
feat(*): acquisition storage

### PR DESCRIPTION
**Pull request by anooparveti**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | ECAN-1674 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

1. Addition of "My Identity Documents" section
2. KYC - Banner to inform customers
3. New page - Upload my Identity files
4. Also tracking

### CODE REVIEW

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function AccountSidebar() {
   const region = environment.getRegion();
   const isEnterprise = environment.getUser()?.enterprise;
 
-  const getAccountSidebar = (availability: Record<string, string> | null) => {
+  const getAccountSidebar = async (availability: Record<string, string> | null) => {
```
Comment (darsene) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx:27]: no need to be async anymore, it's managed with `useEffect`

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx [hunk 2]
```diff
@@ -125,11 +143,11 @@ export default function AccountSidebar() {
     getFeatures,
   );
 
-  const buildMenu = () =>
+  const buildMenu = async () =>
```
Comment (darsene) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx:146]: no need to be async anymore, it's managed with `useEffect`

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx [hunk 3]
```diff
@@ -125,11 +143,11 @@ export default function AccountSidebar() {
...
     Promise.resolve({
       id: 'my-account-sidebar',
       label: '',
-      subItems: [...getAccountSidebar(availability)],
+      subItems: [...(await getAccountSidebar(availability))],
```
Comment (darsene) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx:150]: `await` is not necessary anymore

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx [hunk 4]
```diff
@@ -149,6 +149,29 @@ const Sidebar = (): JSX.Element => {
...
+        let isIdentityDocumentsVisible;
+        if (results['identity-documents']) {
+          const { status } = await reketInstance.get(`/me/procedure/identity`);
+          if (!['required','open'].includes(status)) {
+            isIdentityDocumentsVisible = false;
+          }
+        } else {
+          isIdentityDocumentsVisible = false;
+        }
+        if (!isIdentityDocumentsVisible) {
```
Comment (darsene) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx:165]: There is no case where `isIdentityDocumentsVisible = true`. Mistake ?

**Review by darsene — CHANGES_REQUESTED** [review]

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx (hunk 4 above)
Comment (sachinrameshn) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx:165]: Hi David, By default we have added the identity document option into sidebar and we are removing it when ever we have isIdentityDocumentsVisible has false/undefined. If it is true we wont remove it. Hence there is not action for True case.

**Review by JacquesLarique — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Pull request merged**